### PR TITLE
Promote KEP-2681 to beta in 1.29

### PR DESCRIPTION
**Pull request by wzshiming**
#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Promote PodHostIPs condition to beta.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/issues/2681
- [Usage]: <link>
- [Other doc]:
```

### CODE REVIEW

File: test/e2e_node/pod_host_ips.go [hunk 1]
```diff
@@ -38,9 +37,11 @@ import (
 	e2epodoutput "k8s.io/kubernetes/test/e2e/framework/pod/output"
 	"k8s.io/kubernetes/test/e2e/network/common"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+	netutils "k8s.io/utils/net"
+	"k8s.io/utils/ptr"
 )
 
-var _ = common.SIGDescribe("Dual Stack Host IP [Feature:PodHostIPs]", func() {
+var _ = common.SIGDescribe("DualStack Host IP [NodeFeature:PodHostIPs] [Feature:PodHostIPs]", func() {
```
Comment (wzshiming) [inline test/e2e_node/pod_host_ips.go:44]: https://github.com/kubernetes/test-infra/blob/f8adf234323fa8a9d0c97451b24302a78d4f8c4c/config/jobs/kubernetes/sig-network/sig-network-kind.yaml#L28

https://github.com/kubernetes/test-infra/blob/f8adf234323fa8a9d0c97451b24302a78d4f8c4c/config/jobs/kubernetes/sig-node/containerd.yaml#L165

Add these cases into Testgrid, I'm not sure it's going to work.

**Comment by wzshiming** [comment]
PTAL

/assign @thockin

File: test/e2e_node/pod_host_ips.go [hunk 2]
```diff
@@ -250,9 +209,91 @@ var _ = common.SIGDescribe("Dual Stack Host IP [Feature:PodHostIPs]", func() {
 
 			testDownwardAPI(ctx, f, podName, env, expectations)
 		})
+
+		ginkgo.It("rollback", func(ctx context.Context) {
```
Comment (thockin) [inline test/e2e_node/pod_host_ips.go:213]: "It" statements need to describe what is being tested - tsee the manyu other examples

Comment (wzshiming) [reply test/e2e_node/pod_host_ips.go]: Done

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**